libs/route-config/src/lib/route-tag/route-tag.directive.ts

```diff
@@ -1,55 +1,24 @@
-import { Directive, Input, OnDestroy, OnInit, TemplateRef, ViewContainerRef } from '@angular/core';
```

Comment:
I guess everything still works if the cypress tests ran :)

Reply:
yes